src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs

```diff
@@ -34,11 +33,13 @@
 //      Build Number
 //      Revision
 //
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
-[assembly: InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.UnitTests")]
-[assembly: InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.IntegrationTests")]
-[assembly: InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.Test.Common")]
-[assembly: InternalsVisibleTo("MicrosoftSqlToolsServiceLayer")]
-[assembly: InternalsVisibleTo("MicrosoftKustoServiceLayer")]
-[assembly: InternalsVisibleTo("Microsoft.SqlTools.ManagedBatchParser.IntegrationTests")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.0.0.0")]
+[assembly: AssemblyFileVersion("3.0.0.0")]
+#if NET6_0_OR_GREATER
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.UnitTests")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.IntegrationTests")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.SqlTools.ServiceLayer.Test.Common")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("MicrosoftSqlToolsServiceLayer")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("MicrosoftKustoServiceLayer")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.SqlTools.ManagedBatchParser.IntegrationTests")]
```

Comment:
please just make everything public and get rid of these attributes.
We want to consume this batchparser in other apps and we shouldn't have to guess which "internal" components those other apps might want to use.

Reply:
Glad to know I'm not the only one who thinks that InternalsVisibleTo are just evil...

Reply:
Of course, being public makes one be more diligent at implementing things... in that you need to honor a contract. But, then again, you want to honor the contract even for your "friends" that you grant access to your internals...

Reply:
Can't agree more! https://github.com/microsoft/sqltoolsservice/pull/1747#discussion_r1014346676